packages/mason_cli/lib/src/commands/make.dart

```diff
@@ -85,6 +113,14 @@ class _MakeCommand extends MasonCommand {
     final target = DirectoryGeneratorTarget(Directory(outputDir));
     final disableHooks = results['no-hooks'] as bool;
     final quietMode = results['quiet'] as bool;
+
+    final watch = results['watch'] as bool;
+
+    // TODO(alestiago): Investigate when can the path of a BrickYaml be null.
+    // if (watch && _brick.path == null) {
+    //   usageException('Cannot watch a brick without a path.');
+    // }
```

Comment:
@felangel when would a BrickYaml not have a path (i.e. when can _brick.path be null)?